back/app/models/user.rb

```diff
@@ -427,6 +427,11 @@ def active?
     registered? && !blocked? && !confirmation_required?
   end
 
+  def blank_and_can_be_deleted?
```

Comment:
If we are getting these really long and hard to understand conditionals for certain registrations, I wonder if we should think about (maybe not now) having another user attribute which would be something like registration_type = 'clave_unica' | 'email' | 'google' etc which might make it easier to understand what's going on.

Reply:
I agree, it's getting harder and harder. The only downside of `registration_type` I see is the checks like `if registration_type == 'clave_unica'` are not generic anymore (cannot be used in other methods). But it's an upside too, I agree.

Another solution could be to have sth like a state machine (similar to what we have on the FE).